src/main/java/com/palantir/gradle/versions/FixLegacyJavaConfigurationsPlugin.java

```diff
@@ -94,16 +84,16 @@ private void injectVersions(Configuration conf, GetVersion getVersion) {
                 // don't interfere with the way forces trump everything
                 for (ModuleVersionSelector force : conf.getResolutionStrategy().getForcedModules()) {
                     if (requested.getGroup().equals(force.getGroup()) && requested.getName().equals(force.getName())) {
-                        details.because(String.format(
-                                "Would have recommended a version for %s:%s, but a force is in place",
-                                requested.getGroup(),
-                                requested.getName()));
+                        details.because(
+                                String.format(
+                                        "Would have recommended a version for %s:%s, but a force is in place",
+                                        requested.getGroup(), requested.getName()));
```

Comment:
I made the non-splitting path a bit stricter so now if you have a single call inside this method (`details.because`) i.e. just `String.format()`, it won't try to split it anymore.
We can probably relax this later to allow some cases through.